Library/Formula/ansible.rb

```diff
@@ -156,13 +156,42 @@ class Ansible < Formula
   end
 
   #
-  # boto (for AWS support)
+  # boto/boto3 (for AWS support)
   #
   resource "boto" do
     url "https://pypi.python.org/packages/source/b/boto/boto-2.38.0.tar.gz"
     sha256 "d9083f91e21df850c813b38358dc83df16d7f253180a1344ecfedce24213ecf2"
   end
 
+  resource "boto3" do
```

Comment:
What modules are these new dependencies required for?  If they're ones that should be enabled by default, add `revision 1` underneath line 5 so everyone updates.